libs/admin-app/dashboard/src/lib/dashboard-overview/dashboard-overview.component.ts

```diff
@@ -347,7 +347,7 @@ export class DashboardOverviewComponent implements AfterViewInit
                         {
                             this.TrafficLights[month]++;
                         }
-                        else if (type == "Electricity Outage")
+                        else if (type == "Electricity outage")
```

Comment:
I think it should stay as Water Outage and Electricity Outage, other wise the line chart does not work correctly

Reply:
It did have the capital O's when I was working it. I changed it to this to make them work. I'll pull and check if the changes are unnecessary now though.